src/hyperion/external_interaction/callbacks/ispyb_callback_base.py

```diff
@@ -202,6 +208,7 @@ def _handle_ispyb_transmission_flux_read(self, doc) -> Sequence[ScanDataInfo]:
     def populate_info_for_update(
         self,
         event_sourced_data_collection_info: DataCollectionInfo,
+        event_sourced_position_info: Optional[DataCollectionPositionInfo],
```

Comment:
Related to my earlier comment, if the position info was part of the collection info, we wouldn't need an extra parameter in this base callback. Seems weird to add it to the base callback when only rotations are using it